cli/src/commands/env/init.go

```diff
@@ -0,0 +1,43 @@
+package env
+
+import (
+	"fmt"
+	"horahora/cli/src/lib/errors"
+	"horahora/cli/src/lib/fs"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+)
+
+// `env init`
+var envInit = &cobra.Command{
+	Use:   "init",
+	Short: "Initialize environment variables",
+	PreRun: func(cmd *cobra.Command, args []string) {
+		fmt.Println("Initiallizing environment variables...")
+	},
+	PostRun: func(cmd *cobra.Command, args []string) {
+		fmt.Println("Finished environment variables initialization.")
+	},
+	Run: initEnvVars,
+}
+
+// Copies the `configs/.env.example` file into `.env`
+//
+// @TODO: create `.env` from schema
+func initEnvVars(cmd *cobra.Command, args []string) {
+	workingFolder, pathErr := os.Getwd()
+	errors.CheckError(pathErr)
+
+	envFilePath := filepath.Join(workingFolder, envFileName)
+
+	if fs.IsExist(envFilePath) {
+		fmt.Println("\".env\" file already exists, skipping initialization.")
```

Comment:
could optionally use e.g. logrus with an error-level message here, or a log panic

Reply:
Does it need to be a log? As for now the tool is only for development so there is always going to be a person looking at the console. It's also a broader scope question on how logs and errors should be handled across the entire project. Something more succinct than just returning a tuple from single function.